src/AirtableClient.php

```diff
@@ -69,21 +69,21 @@ public function findOneById(string $table, string $id, ?string $dataClass = null
 
         $recordData = $response->toArray();
 
-        if ($dataClass) {
+        if (null !== $dataClass) {
             $recordData['fields'] = $this->normalizer->denormalize($recordData['fields'], $dataClass);
         }
 
         return AirtableRecord::createFromRecord($recordData);
     }
 
     /**
-     * @inheritdoc
+     * {@inheritdoc}
      */
     public function findTheLatest(string $table, $field, ?string $dataClass = null): ?AirtableRecord
     {
-        $url = $this->id . '/'
-            . $table . '?pageSize=1&sort%5B0%5D%5Bfield%5D='
-            . $field . '&sort%5B0%5D%5Bdirection%5D=desc';
+        $url = $this->id.'/'
+            .$table.'?pageSize=1&sort%5B0%5D%5Bfield%5D='
+            .$field.'&sort%5B0%5D%5Bdirection%5D=desc';
```

Comment:
Use sprintf instead of concatenation ?

Reply:
It was done in my first PR #1 dont understand how we lost that :p

Reply:
Weird

Reply:
Actually these lines were not changed in #1.
I updated the PR with another approach using ``http_build_query` that should improve readability.

Reply:
It was done here : https://github.com/yoanbernabeu/Airtable-Client-Bundle/pull/1/files#diff-4fca284f12d70bb5aeae63b1d3090a19d7d2a047684d5f441cb0d07e41062c24R37

Reply:
Anyway the new way u did it is great :)